docs/resources/azure_ddos_protection_resources.md

```diff
@@ -0,0 +1,95 @@
+---
+title: About the azure_ddos_protection_resources Resource
+platform: azure
+ ---
+
+# azure_ddos_protection_resources
+
+Use the `azure_ddos_protection_resources` InSpec audit resource to test properties related to bastion hots for a resource group or the entire subscription.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+
+Also, refer to [Azure documentation](https://docs.microsoft.com/en-us/rest/api/virtualnetwork/ddos-protection-plans/list) for  properties available.
+Any attribute in the response may be accessed with the key names separated by dots (`.`).
+## Syntax
+
+An `azure_ddos_protection_resources` resource block returns all Azure bastion hots, either within a Resource Group (if provided)
+ ```ruby
+ describe azure_ddos_protection_resources(resource_group: 'my-rg') do
+   
+ end
+ ```
+
+## Properties
+
+|Property       | Description                                                                          | Filter Criteria<superscript>*</superscript> |
+ |---------------|--------------------------------------------------------------------------------------|-----------------|
+| name           | A list of the unique resource names.                                                | `name`            |
+| ids            | A list of ddos_protection_plan ids .                                                       | `id`              |
+| virtual_networks           | The list of virtual networks associated with the DDoS protection plan resource. This list is read-only.| `virtual_network`             |
```

Comment:
as per [this](https://github.com/inspec/inspec-azure/pull/415/files#diff-e5245bbcb7b027855cfdee2d684821191de17744f67f370beb77b710ee5e1bd0R36) field should be `virtual_networks`